Assets/_BForBoss/_Weapons/Scripts/Equipment/EquipmentBehaviour.IEquipmentData.cs

```diff
@@ -1,27 +1,24 @@
-using FMODUnity;
-
 namespace Perigon.Weapons
 {
     public interface IEquipmentData
     {
         int AmmunitionAmount { get; }
         int MaxAmmunitionAmount { get; }
+        
         string NameOfWeapon { get; }
         float MaxReloadDuration { get; }
         float ElapsedReloadDuration { get; }
-        EventReference WeaponShotAudio { get; }
     }
 
     public partial class EquipmentBehaviour: IEquipmentData
     {
-        public int AmmunitionAmount => CurrentWeapon.AmmunitionAmount;
-        public int MaxAmmunitionAmount => CurrentWeapon.MaxAmmunitionAmount;
-        public string NameOfWeapon => CurrentWeapon.NameOfWeapon;
+        public int AmmunitionAmount => 0;
```

Comment:
A lot of these values will be gone soon. So i'm just putting placeholder values for now.